nemo/collections/asr/models/ctc_models.py

```diff
@@ -106,6 +106,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Adapter modules setup (from ASRAdapterModelMixin)
         self.setup_adapters()
 
+        # setting up interCTC loss (from InterCTCMixin)
```

Comment:
Probably best to keep adapter setup for last, move it up by one level. Btw in a later PR we should update the execution flow diagram for CTC models - https://github.com/NVIDIA/NeMo/tree/main/examples/asr/asr_ctc#model-execution-overview